src/bci_build/package.py

```diff
@@ -975,7 +975,7 @@ async def write_file_to_dest(fname: str, contents: Union[str, bytes]) -> None:
                     write_file_to_dest(
                         changes_file_name,
                         f"""-------------------------------------------------------------------
-{datetime.datetime.now(tz=datetime.timezone.utc).strftime("%a %b %d %X %Z %Y")} - SUSE Update Bot <noreply@suse.com>
+{datetime.datetime.now(tz=datetime.timezone.utc).strftime("%a %b %d %X %Z %Y")} - SUSE Update Bot <dcermak@suse.com>
```

Comment:
That's just misleading now (you're not an update bot)

How about simply using the team mailing list?